### ISSUE TITLE
unable to resolve cucumber support files path relative to project

### ISSUE THREAD

**Issue by Mouad-B** [cucumber/cucumber-js#1129]
I'm using Cucumber v5 on linux to do some e2e tests but I can't require support files with relative path to the project:

> node_modules
> cucumber
> ├── features
> ├── step_definitions

I run test using this command `./node_modules/.bin/cucumber-js cucumber/features --require cucumber/step_definitions`
Cucumber is able to list files in `cucumber/step_definitions` but can't require them: 

> Error: Cannot find module 'cucumber/step_definitions/stepFile.js'

When I changed line `163 `of `/node_modules/cucumber/lib/cli/index.js`:
`return require(codePath);`
with:
`return require(process.cwd()+'/'+codePath);`
everything works correctly. Can anyone tell me what I'm missing here ?

**Comment by charlierudolph** [comment]
Very odd. We are using the `absolute` option of glob so it should be resolving the paths to absolute paths. Can you play around with glob and see if that option works as expected on your machine?

Side note: Do you not like the standard directory structure with features/ as the top level and step definitions nested inside?

**Comment by Mouad-B** [comment]
Thank you for your answer
I tried to use glob to require files as used by the module and I get exactly the same behaviour:
```
const glob = require("glob");

glob("cucumber/test_*.js", {absolute: true}, (error, files) => {
    if (error) {
        console.log({error});
    } else {
        console.log({files})
        files.forEach(file => {
            require(file);
        });
    }
});
```
I get the following:

> { files: [ 'cucumber/test_1.js' ] }
> Error: Cannot find module 'cucumber/test_1.js'

So you think it is more a glob issue rather than cucumber ?

**Comment by Mouad-B** [comment]
using {realpath: true} instead of {absolute: true} also resolves the issue

**Comment by charlierudolph** [comment]
Yeah, I'm currently thinking this is an issue with glob. Can you create a tiny repo that can be used to reproduce the error just so I can test it as well? Also what OS are you on?

**Comment by Mouad-B** [comment]
That's really weird.. When I created a tiny project as suggested with cucumber, everything works as expected.. Before closing the issue, do you know of any reason that would make glob not use absolute option correctly ?

**Comment by ngerritsen** [comment]
So I have the exact same issue. My findings:

- I have two separate projects, they both use cucumber 4.2.1. Cucumber depends on `glob ^7.0.0`
- The __older__ one has resolved to `glob 7.1.2` a few months ago, that version is now in the package-lock.
- The __newer__ one somehow resolves to `glob 7.0.5` today, which is weird because that is an older version, but it breaks because in this version `absolute: true` seems to break.

We could maybe solve this by letting cucumber depend on `glob ^7.1.2`?

**Comment by Mouad-B** [comment]
Thanks you @ngerritsen for sharing this ! It solved my issue with the absolute path
